### PR TITLE
notes for Prometheus integration

### PR DESCRIPTION
**Pull request by rigazilla**
steps for integrating Infinispan and Prometheus.
Needs #286

### CODE REVIEW

**Comment by rigazilla** [comment]
@oraNod, some notes about Prometheus

File: documentation/asciidoc/stories/prometheus_integration.adoc [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+#### Deploy the service monitor
+A service monitor is a customer resource (ServiceMonitor) that represent the application to be monitored.
+The Prometheus Operator, when a ServiceMonitor cr is deployed to the system, genetares all the configuration needed to scrap the application.
+The information needed to define a ServiceMonitor are:
+
+1. how to select the workload to scrap. This will be represented by a selector item;
+2. how to connect to selected workload. This will be represented by an endpoint item.
+
+The ServiceMonitor template to populate is the following:
```
Comment (Crumby) [inline documentation/asciidoc/stories/prometheus_integration.adoc]: Is this example intentionally "compressed"? It's missing `kind` and `metadata`.

File: documentation/asciidoc/stories/prometheus_integration.adoc [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+     scheme: https
+     tlsConfig:
+       insecureSkipVerify: true
+       serverName: <certificate CN>
+ namespaceSelector:
+   matchNames:
+     - <Infinispan namespace>
+ selector:
+   matchLabels:
+     app: infinispan-service
```
Comment (Crumby) [inline documentation/asciidoc/stories/prometheus_integration.adoc]: If I'm reading #286 correctly shouldn't this be `<InfinispanCRName>: infinsipan-service`? Also won't this match all the cluster that will possibly run in the same namespace? How about adding `cluster: <InfinispanCRName>`? If I'm not mistaken then all the resources created by Operator get this label...

Comment (rigazilla) [reply documentation/asciidoc/stories/prometheus_integration.adoc]: yeah oversight, we need to identify the cluster. Thanks!

File: documentation/asciidoc/stories/prometheus_integration.adoc (hunk 1 above)
Comment (rigazilla) [reply documentation/asciidoc/stories/prometheus_integration.adoc]: bad copy/paste. thanks!

File: documentation/asciidoc/stories/prometheus_integration.adoc (hunk 2 above)
Comment (Crumby) [reply documentation/asciidoc/stories/prometheus_integration.adoc]: Btw, reading it one more time after myself you can ignore the second part of the comment as it's invalid. `<InfinispanCRName>: infinsipan-service` should be enough for the verification. Second applies only if the we would be matching `app: infinispan-service`.

File: documentation/asciidoc/stories/prometheus_integration.adoc (hunk 1 above)
Comment (rigazilla) [reply documentation/asciidoc/stories/prometheus_integration.adoc]: done

File: documentation/asciidoc/stories/prometheus_integration.adoc (hunk 2 above)
Comment (rigazilla) [reply documentation/asciidoc/stories/prometheus_integration.adoc]: done

**Pull request closed without merging**